tigers-lottery/app/src/main/java/com/example/tigers_lottery/DatabaseHelper.java

```diff
@@ -71,14 +65,67 @@ public void onEvent(@Nullable QuerySnapshot value, @Nullable FirebaseFirestoreEx
                 });
     }
 
```

Comment:
Add a helper method to grab event by organizer id and use it display hosted events by that organizer